ChangeLog

```diff
@@ -1,5 +1,13 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix Renaming "new" variable #1783.
+     compile error(new variable) with arm-none-eabi-gcc(c++) on mbed TLS 2.7.0.
```

Comment:
I would change this sentence to:
~~~
Fix compilation error on c++, because of a variable named new. Found and fixed by Hirotaka Niisato in #1783 
~~~
Unless you want to credit yourself with your github name.

Reply:
Thank you, I'm okay "Hirotaka Niisato" .